DigitalLearningSolutions.Data/Models/Courses/DelegateCourseProgressInfo.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DigitalLearningSolutions.Data.Models.Progress;
+using System.Collections.Generic;
+
+namespace DigitalLearningSolutions.Data.Models.Courses
+{
+    public class DelegateCourseProgressInfo : DelegateCourseInfo
+    {
+        public string CentreName { get; set; }
+        public string CandidateName { get; set; }
+        public string Course { get; set; }
+        public int DiagnosticAttempts { get; set; }
+        public int TotalTime { get; set; }
+        public int LearningDone { get; set; }
+        public int PLAttempts { get; set; }
+        public int PLPasses { get; set; }
+        public int Sections { get; set; }
+        public bool IsAssessed { get; set; }
```

Comment:
Please address this build warning as we are now moving towards addressing these as per the jira issue https://hee-tis.atlassian.net/browse/TD-2316

Reply:
Done.